explorer/src/Pos/Explorer/Txp/Global.hs

```diff
@@ -65,16 +65,16 @@ applyBlund txpBlund = do
     -- use e. g. SlotId epoch minBound. If it's Right, you can use headerSlotL lens.
     --
     -- type TxpBlund = (TxpBlock, TxpUndo)
-    -- type TxpBlock = Either (Some IsGenesisHeader) (Some IsMainHeader, TxPayload)
+    -- type TxpBlock = ComponentBlock TxPayload
 
     let txpBlock = txpBlund ^. _1
     let slotId   = case txpBlock of
-            Left genesisBlock -> SlotId
+            ComponentBlockGenesis genesisBlock -> SlotId
                                   { siEpoch = genesisBlock ^. epochIndexL
                                   , siSlot  = minBound
                                   -- ^ Genesis block doesn't have a slot, set to minBound
                                   }
-            Right mainBlock   -> mainBlock ^. _1 . headerSlotL
+            ComponentBlockMain mainBlock _  -> mainBlock ^. headerSlotL
```

Comment:
AFAIU, `mainBlock` is now `Some IsMainHeader`, so I think it should be called `mainHeader`?